jest_after_env_setup.js

```diff
@@ -18,3 +18,19 @@ import {configure} from "enzyme";
 import Adapter from "enzyme-adapter-react-16";
 
 configure({adapter: new Adapter()});
+
+// Worker
```

Comment:
It took me a while to figure out why you added this. But, similar to how you added `jest-canvas-mock`, let's also add https://github.com/developit/jsdom-worker. Seems to be not super active, but better than these mocks.

(See also this comment: https://github.com/jsdom/jsdom/issues/1721#issuecomment-1079407658)